frontend/packages/dev-console/integration-tests/features/topology/quick_search_from_topology_views _to_add_to_project_ODC-2925

```diff
@@ -0,0 +1,73 @@
+Feature: Provide quick search from topology/list views to add to project 
+    As a user, when in Topology List/Graph view, I should be able to have a quick way to search for items to add to my app/project
+
+Background:
+    Given user is at developer perspective
+    And user is "aut-quick-search" namespace
+
+
+@regression
+Scenario: Quick search button in topology chart view
+    Given user is at topology view
+    When user clicks on chart view button
+    Then user can see quick search button
+
+
+@regression
+Scenario: Quick search button in topology list view
+    Given user is at topology view
+    When user clicks on list view button
+    Then user can see quick search button
+
+
+@regression
+Scenario: Quick search bar in topology chart view
+    Given user is at topology chart view
+    When user clicks Add to project button
+    Then user can see Add to project search bar
```

Comment:
So above you have mentioned it as quick search button and here you are adding Add to project button. Keep it same I would suggest.

Reply:
I think Add to project button and Add to project search bar is more appropriate as UX defines it that way

Reply:
Np. Just keep it same.